test/abstract_syntax_tree_location_data.coffee

```diff
@@ -3928,13 +3928,13 @@ test "AST as expected for If node", ->
             line: 1
             column: 17
       ]
-      start: 8
+      start: 9
```

Comment:
Making the change to `then` location data handling in the rewriter actually made some of these existing tests including `then` "more correct": rather than the block location data starting a character before the `then` (which I'd pointed out as perhaps slightly unexpected but not problematic), it now starts on the first character of `then`